src/ui/public/courier/__tests__/saved_object.js

```diff
@@ -82,19 +83,108 @@ describe('Saved Object', function () {
     const savedObject = new SavedObject(config);
     return savedObject.init();
   }
-
-  beforeEach(ngMock.module('kibana'));
-  beforeEach(ngMock.inject(function (es, Private) {
+  beforeEach(ngMock.module('kibana',
+
+    // The default implementation of safeConfirm uses $timeout which will cause
+    // the test environment to hang.
+    function ($provide) {
+      $provide.decorator('safeConfirm', () => {
+        return (message) => {
```

Comment:
"If your arrow function is only returning an object literal, then wrap the object in paranthesis rather than using an explicit return"

https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#arrow-functions